### PR TITLE
add hello.py

### PR DESCRIPTION
**Pull request by r6lok**
add hello.py

use to do github demo

### CODE REVIEW

File: python/hello.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def main():
+    print("hello world!")
+
+if __name__ == "__main__":
+    main()
```
Comment (jnie1026) [inline python/hello.py:5]: missing new line at the end of the code

File: python/hello.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def main():
+    print("hello world!")
```
Comment (jnie1026) [inline python/hello.py:2]: take an put of person's name

**Comment by r6lok** [comment]
All changes have been updated

**Review by jnie1026 — APPROVED** [review]
Looks good to me. Approved

**Pull request merged**